packages/amplify-graphql-schema-generator/src/utils/vpc-helper.ts

```diff
@@ -41,7 +41,7 @@ import * as fs from 'fs-extra';
 import ora from 'ora';
 import { printer } from '@aws-amplify/amplify-prompts';
 
-const DB_ENGINES = ['aurora-mysql', 'mysql'];
+const DB_ENGINES = ['aurora-mysql', 'mysql', 'postgres', 'aurora-postgresql'];
```

Comment:
Any reason we're not parallel between "postgres" and "aurora-postgres**ql**"? If these are supposed to match a source elsewhere (e.g., supported config values in RDS) we should document that so we've got a reference